src/Windows/Avalonia.Win32.Automation/AutomationNode.RangeValue.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Avalonia.Automation.Provider;
+using UIA = Avalonia.Win32.Interop.Automation;
+
+namespace Avalonia.Win32.Automation
+{
+    internal partial class AutomationNode : UIA.IRangeValueProvider
+    {
+        double UIA.IRangeValueProvider.Value() => InvokeSync<IRangeValueProvider, double>(x => x.Value);
```

Comment:
Should we add `Get` or `Set` prefix to conform the naming rules?

Reply:
I did that, but then reverted trying to keep diff smaller. In the end, git still couldn't connect old and new files.

Reply:
Might just return these Get/Set as well